p2p/src/error.rs

```diff
@@ -188,14 +195,17 @@ impl BanScore for ProtocolError {
         match self {
             ProtocolError::DifferentNetwork(_, _) => 100,
             ProtocolError::InvalidVersion(_, _) => 100,
-            ProtocolError::InvalidMessage => 100,
             ProtocolError::Unresponsive => 100,
             ProtocolError::LocatorSizeExceeded(_, _) => 20,
             ProtocolError::BlocksRequestLimitExceeded(_, _) => 20,
             ProtocolError::HeadersLimitExceeded(_, _) => 20,
-            ProtocolError::UnknownBlockRequested => 20,
+            ProtocolError::UnknownBlockRequested(_) => 20,
+            ProtocolError::DuplicatedBlockRequest(_) => 20,
             ProtocolError::DisconnectedHeaders => 20,
             ProtocolError::UnexpectedMessage(_) => 20,
+            ProtocolError::ZeroBlocksInRequest => 20,
+            ProtocolError::HandshakeExpected => 20,
```

Comment:
Is 20 intended in all of these? It sounds OK for ZeroBlocks... but the others, the handshake thing sounds bad. Also limit breaking sounds bad too. Maybe they deserve 100?